cypress/integration/pages/articles/tests.js

```diff
@@ -130,7 +130,7 @@ export const testsThatFollowSmokeTestConfig = ({
         });
       });
 
-      if (serviceHasInlineLink(service) && Cypress.env('APP_ENV') === 'local') {
+      if (serviceHasInlineLink(service)) {
```

Comment:
I realised we need this to not run on live as the live links do not always have articles inline links. So I changed this line, and you will see it in my commit.